src/google/Line.js

```diff
@@ -100,119 +100,123 @@
     Line.prototype.getChartOptions = function () {
         var retVal = CommonND.prototype.getChartOptions.apply(this, arguments);
 
-        retVal.lineWidth = this._lineWidth;
-        retVal.lineDashStyle = this._gloabLineDashStyle;
-        retVal.orientation = this._orientation;
-        retVal.pointShape = this._globalPointShape;
-        retVal.pointSize = this._globalPointSize;
+        retVal.lineWidth = this.lineWidth();
+        retVal.lineDashStyle = this.gloabLineDashStyle();
+        retVal.orientation = this.orientation();
+        retVal.pointShape = this.globalPointShape();
+        retVal.pointSize = this.globalPointSize();
         
         retVal.hAxis = {};
         retVal.vAxis = {};
         
         // hAxis
-        retVal.hAxis.baseline = this._hAxisBaseline;
-        retVal.hAxis.baselineColor = this._hAxisBaselineColor;
-        retVal.hAxis.direction = this._hAxisDirection;
+        retVal.hAxis.baseline = this.hAxisBaseline();
+        retVal.hAxis.baselineColor = this.hAxisBaselineColor();
+        retVal.hAxis.direction = this.hAxisDirection();
         retVal.hAxis.gridlines = {
-            count: this._hAxisGridlinesCount,
-            color: this._hAxisGridlinesColor
+            count: this.hAxisGridlinesCount(),
+            color: this.hAxisGridlinesColor()
         }
         retVal.hAxis.minorGridlines = {
-            count: this._hAxisMinorGridlinesCount,
-            color: this._hAxisMinorGridlinesColor
+            count: this.hAxisMinorGridlinesCount(),
+            color: this.hAxisMinorGridlinesColor()
         }        
-        retVal.hAxis.logScale = this._hAxisLogScale;
-        retVal.hAxis.textPosition = this._hAxisTextPosition;
-        retVal.hAxis.title = this._hAxisTitle;
-        retVal.hAxis.minValue = this._hAxisMinValue;
-        retVal.hAxis.maxValue = this._hAxisMaxValue;
+        retVal.hAxis.logScale = this.hAxisLogScale();
+        retVal.hAxis.textPosition = this.hAxisTextPosition();
+        retVal.hAxis.title = this.hAxisTitle();
+        retVal.hAxis.minValue = this.hAxisMinValue();
+        retVal.hAxis.maxValue = this.hAxisMaxValue();
         
-        retVal.hAxis.hAxisAllowContainerBoundaryTextCufoff = this._hAxisAllowContainerBoundaryTextCufoff;
-        retVal.hAxis.slantedText = this._hAxisSlantedText;
-        retVal.hAxis.slantedTextAngle = this._hAxisSlantedTextAngle;
-        retVal.hAxis.maxAlternation = this._hAxisMaxAlternation;
-        retVal.hAxis.maxTextLines = this._hAxisMaxTextLines;
-        retVal.hAxis.minTextSpacing = this._hAxisMinTextSpacing;
+        retVal.hAxis.hAxisAllowContainerBoundaryTextCufoff = this.hAxisAllowContainerBoundaryTextCufoff();
```

Comment:
its suppose to be spelled Cutoff .. the publish param might be spelled wrong too